### ISSUE TITLE
Using our own models

### ISSUE THREAD

**Issue by Woisek** [Linaqruf/sd-notebook-collection#21]
Any chance, to get the possibility, to use a checkpoint from google drive itself and not been limited to this few? 🙂

**Comment by BinaryQuantumSoul** [comment]
Please give context. I see you're coming from vladmantic/automatic so here's how to do in the fast-vladmantic-webui:
![image](https://user-images.githubusercontent.com/19613657/235766528-d52d6179-b510-4699-b539-6f01c240e66a.png)
Just paste the url there xD

**Comment by Woisek** [comment]
Ah, cool, thanks. Haven't noticed it ... 🤪

**Comment by sahildeshpande** [comment]
Can I paste a drive path instead of a URL?

**Comment by BinaryQuantumSoul** [comment]
> Can I paste a drive path instead of a URL? 

Yes you can if you mount the drive in the first cell of the notebook. Just use /content/drive/MyDrive/yourfilepath you cannot give a directory sadly, just an individual file

**Comment by Linaqruf** [comment]
@BinaryQuantumSoul Thank you for the answer! I appreciate it. And sorry for my late reply.
Yes, you can use your own model from google drive by doing this step
1. Set `mount_drive` to `True`
2. Copy the model path from you google drive, and paste the path to `custom_model_url` in **Custom Download Corner** cell
3. Run cell
You can also use your own LoRA, embeddings, hypernetwork, and even Install custom extensions too! By doing the same step, but paste the path/url to respective category url.

**Comment by Linaqruf** [comment]
**[Advanced]**

You can also load your whole model folder instead of copying every model path. I have integrated `unionfs-fuse` inside the notebook to merge your own model folder.

Please do this step:
1. Set `mount_drive` to `True`
2. Copy the model **folder** path from your google drive
3. Add prefix `fuse:` before your model folder path, example
```
fuse:/content/drive/MyDrive/my-model
```
4. Run the cell

You can load your model from google drive at once!

**Comment by BinaryQuantumSoul** [comment]
Thank you, I didn't know about fuse, I had to copy path ten times.

**Comment by Linaqruf** [comment]
I'm sorry I renamed this issue, hope it will help other friends out there.
